public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -36,6 +36,23 @@ import { setupKeyboardShortcuts } from './keyboardShortcuts';
 export interface DashboardSceneState extends SceneObjectState {
   /** The title */
   title: string;
+  /** The description */
+  description?: string;
+  /** The tags */
+  tags?: string[];
+  /** Is editable */
+  editable?: boolean;
+  /** Default Timezone */
+  timezone: string;
+  /** Day when the week starts */
+  weekStart: string;
+  /** True when live */
+  liveNow: boolean;
+  timepicker: {
+    refresh_intervals: string[];
+    nowDelay: string;
+    hidden: boolean;
+  };
```

Comment:
I don't think we need to have this as a state of DashbaordScene. Keep in mind that different aspects of a dashboard are expressed as atomic scene objects in new architecture. Have a look at https://github.com/grafana/grafana/blob/main/public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts#L220 to see what concepts are abstracted to a scene object. 

With this in mind we should modify corresponding objects state instead of introducing a big "global" state to a DashboardScene

Reply:
@ivanortegaalba @dprokop yes, as dominik said, most of this state already exists on SceneTimeRange (timezone, weekstart) 

liveNow, I think we need to look at that feature and implement that feature before knowing where state should be, same with nowDelay. 

hidden time picker is interesting, now sure. Maybe something to discuss :)

Reply:
> hidden time picker is interesting, now sure. Maybe something to discuss :)

Yeah, this is literally modifying controls on the DashboardScene. I think, given we now have `DashboardControls` abstraction, we could controll visibility via state of this object https://github.com/grafana/grafana/blob/main/public/app/features/dashboard-scene/scene/DashboardControls.tsx#L15

Reply:
Great! Thanks for the feedback! I was working on the premise of trying to serialize the state in a similar way to the Dashboard Model. But, you're totally right, this new way of thinking implies moving the state to the right place, thanks!

Reply:
I moved there, but it implied things like:
- When the dashboard detects changes, it has to look for specific controls like RefreshPicker. This scene object represents more a control than a dashboard scene state. It looks a bit weird. Take a look at the PR and comment :) 
- When serializing, we need to check the controls and serialize the state to move them to the `Dashboard` to be stored 
- For things like `timezone` it is different from the default state than the current one. Otherwise, any current state change will be serialized as the new default. 

The PR is updated, with this approach, feel free to drop some comments